### DISCUSSION TITLE
Sentinel 1.8 平台添加流控规则没有反应，后台也不报错,有哪位大神知道

### DISCUSSION THREAD

**Discussion by sahnggq** [alibaba/Sentinel#2213] [Q&A]
Sentinel 平台添加流控规则没有反应，后台也没有报错，
com.alibaba.csp ：sentinel-apache-dubbo-adapter 版本用的是1.7.x 的版本，1.8.x 也不行。
Spring-boot-starter ： 2.1.6.RELEASE 版本

**Comment by sczyh30** [comment]
Please refer [FAQ](https://github.com/alibaba/Sentinel/wiki/FAQ#q-控制台正确的规则推送后看不到)

**Reply by Liujinixn** [reply]
假设你的接口是 /test/page  ，如果路径中出现了 // 如，//test/java 或者  /test//java 或者 test/java   等等情况 就不行
